CMakeLists.txt

```diff
@@ -47,6 +47,10 @@ if(NOT CMAKE_CXX_FLAGS_RELEASE MATCHES "-O")
   set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} -O3")
 endif()
 
+if(APPLE)
+  set(CMAKE_PREFIX_PATH /opt/pkg)
```

Comment:
I think it's because I installed cmake through homebrew, so it gives me some path issue. I just deleted it and now it works, I will fix it soon.